css-conditional-5/Overview.bs

```diff
@@ -1067,6 +1087,159 @@ Style Container Features</h3>
 	are [=computed value|computed=] with respect to the [=query container=],
 	the same as other values.
 
+<h3 id="scroll-state-container">
+Scroll State Container Features</h3>
+
+	A <dfn export>container scroll-state query</dfn> allows to query a container for various
+	state that depends on scroll position. The ''scroll-state'' [=query container=] can either
+	be the scroller itself, or an element that is affected by the scroll position of an ancestor
+	[=scroll container's=] [=scrollport=].
+
+<h4 id="updating-scroll-state">
+Updating Scroll State</h4>
+	<div class=note>
+		Note: This section is subject to change as a result of resolving
+		<a href="https://github.com/w3c/csswg-drafts/issues/10796">issue #10796</a>
+	</div>
+
+	Scroll state may cause layout cycles since queried scroll state may cause style changes,
+	which may lead to scroll state changes as a result of layout. The same issue exists for
+	[=scroll progress timelines=], and scroll state is handled in a similar manner.
+
+	To avoid such layout cycles, ''scroll-state'' [=query containers=] update their
+	current state once as the last step of [=run the scroll steps=]. Then, after the
+	resizeObserver loop in the
+	<a href="https://html.spec.whatwg.org/multipage/webappapis.html#event-loop-processing-model">HTML event loop processing model</a>,
+	the scroll state of every ''scroll-state'' [=query container=] is updated.
+	If that state has changed since the scroll state update in [=run the scroll steps=],
+	re-run the style and layout update if necessary.
+
+<h4 id="stuck">
+Sticky positioning: the '@container/stuck' feature</h4>
+
+	<pre class="descdef mq">
+		Name: stuck
+		For: @container
+		Value: top | right | bottom | left | block-start | inline-start | block-end | inline-end
+		Type: discrete
+	</pre>
+
+	The '@container/stuck' [=container feature=] queries whether a
+	''position/sticky'' positioned container is visually shifted to stay inside
+	the [=sticky view rectangle=] for the given edge. The logical edges map to
+	physical based on the direction and writing-mode of the [=query container=].
+
+	In the boolean context, the query matches if visual shift is applied in any
+	direction.
+
+	<dl dfn-type=value dfn-for="@container/stuck">
+		<dt><dfn>top</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the top edge.
+		<dt><dfn>right</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the right edge.
+		<dt><dfn>bottom</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the bottom edge.
+		<dt><dfn>left</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the left edge.
+		<dt><dfn>block-start</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the block-start edge.
```

Comment:
link the logical values to their [definitions](https://drafts.csswg.org/css-writing-modes-4/#logical-directions)?